lib/rucio/common/utils.py

```diff
@@ -611,16 +611,13 @@ def default(self, obj):  # pylint: disable=E0202
         return json.JSONEncoder.default(self, obj)
 
 
-def render_json(**data):
-    """ JSON render function
+def render_json(list_=None, **data: Optional[Any]):
```

Comment:
I would rewrite the function signature like this:

```suggestion
def render_json(list_: Optional[list[Any] = None, **data) -> str:
```

`data` is a `dict`

Reply:
Will commit together with the changes defined below :)